inst/schema/report_run_response.json

```diff
@@ -0,0 +1,11 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "type": "object",
+    "properties": {
+        "job_id": {
+            "type": "string"
+        }
+    },
+    "required": ["job_id"],
```

Comment:
any reason not to return as a string rather than an object?  if an object any reason not to use `jobId` as that's probably closer to what Packit uses? And any reason we can't use `task` not `job` as that's what rrq uses?

Reply:
no reason other than i was speculating that we may want to add other things that return from this endpoint, and having an object returned means an easier future refactor in kotlin if another field is needed

yep this can definitely be camelcase

job_id was only used because I believe the sequence diagram uses job id and that percolated through to the tickets, i can change it to task!

Reply:
have changed to `taskId`